whipper/program/cdparanoia.py

```diff
@@ -192,16 +192,17 @@ def getTrackQuality(self):
         reads = self.reads
         logger.debug('getTrackQuality: frames %d, reads %d' % (frames, reads))
 
-        # don't go over a 100%; we know cdparanoia reads each frame at least
-        # twice
         try:
+            # don't go over a 100%
+            # we know that cdparanoia reads each frame at least twice
             return min(frames * 2.0 / reads, 1.0)
         except ZeroDivisionError:
-            return 0
-
+            raise RuntimeError("cdparanoia couldn't read any frame "
```

Comment:
"frame" or "frames"? If it's meant to say "cdparanoia read 0 frames", then it should probably be in plural here.

Reply:
I don't know: the plural form is probably more common so I'll switch to that.